docs/sources/datasources/tempo.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Tempo"
+description = "High volume, minimal dependency trace storage. OSS tracing solution from Grafana Labs."
+keywords = ["grafana", "tempo", "guide", "tracing"]
+type = "docs"
+aliases = ["/docs/grafana/latest/features/datasources/tempo"]
+[menu.docs]
+name = "Tempo"
+parent = "datasources"
+weight = 800
++++
+
+# Tempo data source
+
+Grafana ships with built-in support for Tempo a high volume, minimal dependency trace storage, OSS tracing solution from Grafana Labs. Just add it as a data source, and you are ready to query your traces in [Explore]({{< relref "../explore/index.md" >}}).
+
+## Adding the data source
+To access Tempo settings, click the **Configuration** (gear) icon, then click **Data Sources**, and then click **Tempo**.
```

Comment:
To access Tempo settings, click **Configuration** (gear) icon, then click **Data Sources** > **Tempo**.